client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -209,49 +229,113 @@ class CancelPurchase extends Component {
 					purchaseId={ this.props.purchaseId }
 				/>
 
-				<HeaderCake
-					backHref={ this.props.getManagePurchaseUrlFor(
-						this.props.siteSlug,
-						this.props.purchaseId
-					) }
+				<Button
+					compact
+					borderless
+					className="cancel-purchase__back-link"
+					href={ this.props.getManagePurchaseUrlFor( this.props.siteSlug, this.props.purchaseId ) }
 				>
-					{ titles.cancelPurchase }
-				</HeaderCake>
-
-				<Card className="cancel-purchase__card">
-					<h2>{ heading }</h2>
-
-					<CancelPurchaseRefundInformation
-						purchase={ purchase }
-						isJetpackPurchase={ isJetpackPurchase }
-						includedDomainPurchase={ this.props.includedDomainPurchase }
-						confirmBundledDomain={ this.state.confirmCancelBundledDomain }
-						cancelBundledDomain={ this.state.cancelBundledDomain }
-						onCancelConfirmationStateChange={ this.onCancelConfirmationStateChange }
-					/>
-				</Card>
-
-				<PurchaseSiteHeader siteId={ siteId } name={ siteName } domain={ siteDomain } />
-				<CompactCard className="cancel-purchase__product-information">
-					<div className="cancel-purchase__purchase-name">{ purchaseName }</div>
-					<div className="cancel-purchase__description">{ purchaseType( purchase ) }</div>
-					<ProductLink purchase={ purchase } selectedSite={ this.props.site } />
-				</CompactCard>
-				<CompactCard className="cancel-purchase__footer">
-					<div className="cancel-purchase__refund-amount">
-						{ this.renderFooterText( this.props ) }
+					<Gridicon icon="chevron-left" size={ 18 } />
+					{ this.props.translate( 'Back' ) }
+				</Button>
+
+				<FormattedHeader
+					className="cancel-purchase__formatter-header"
+					brandFont
+					headerText={ heading }
+					align="left"
+				/>
+
+				<div className="cancel-purchase__layout">
+					<div className="cancel-purchase__layout-col cancel-purchase__layout-col-left">
+						<PurchaseSiteHeader siteId={ siteId } name={ siteName } domain={ siteDomain } />
+
+						<CancelPurchaseRefundInformation
+							purchase={ purchase }
+							isJetpackPurchase={ isJetpackPurchase }
+							includedDomainPurchase={ this.props.includedDomainPurchase }
```

Comment:
`includedDomainPurchase` is not a prop that can be passed to `CancelPurchaseRefundInformation`; it's automatically set by the `mapStateToProps` function in that component based on the `purchase` prop. So this line should be removed or (if providing it is necessary), `CancelPurchaseRefundInformation` should be modified to allow setting it as a prop.